src/components/DayBoxSchedule.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from "react";
+import List, { ListItem } from "material-ui/List";
+import Typography from "material-ui/Typography";
+class DayBoxSchedule extends Component {
+  displayClasses() {
+    let card = [];
+    let divKey = 0;
+    if (
+      this.props.calendarMeeting[this.props.month] === undefined ||
+      this.props.calendarMeeting[this.props.month][this.props.day] === undefined
+    ) {
+      ++divKey;
+      card.push(<div key={"noclasses" + divKey} aria-label="No classes" />);
+    } else {
+      if (
```

Comment:
I don't think that this check is necessary - it's a repeat of the previous if statement